source/gear/herbbasket-base.js

```diff
@@ -16,7 +17,7 @@ module.exports = new GearTemplate("Herb Basket",
 		if (user.crit) {
 			pendingHerbCount *= critBonus;
 		}
-		const randomHerb = rollableHerbs[user.roundRns[`${gearName}${SAFE_DELIMITER}herbs`][0] % rollableHerbs.length];
+		const randomHerb = rollableHerbs[user.roundRns[`${module.exports.name}${SAFE_DELIMITER}herbs`][0] % rollableHerbs.length];
```

Comment:
Good fix on these. I'm actually very surprised this works: I distinctly remember being disappointed taking properties directly off of `module.exports` was giving me `undefined`, but it definitely seems to be working fine now. This now working means we should probably do a pass on all the gear to comply with this pattern on all the properties.